src/scss/skin-super-modern/components/_uicontainer.scss

```diff
@@ -34,7 +34,7 @@
     .#{$prefix}-ui-hugeplaybacktogglebutton {
       &:focus {
         box-shadow: inset -4px -3px 2px 9px $color-focus;
-
+        
```

Comment:
please revert this change since it seems unnecessary

Reply:
this was just a linter warning fix